commit 24de0c1d512a323264b56f4e0a664208a6627754
Author: Volker Krause <vkrause@kde.org>
Date:   Fri Feb 15 19:21:01 2019 +0100

    Namespace find_file result
    
    KCoreAddons uses the same name, which causes this to use the cached result
    for the KCoreAddons find_file call, making this fail when KCoreAddons is
    present.

diff --git a/KF5AuthConfig.cmake.in b/KF5AuthConfig.cmake.in
index 0895e60..0bb15ff 100644
--- a/KF5AuthConfig.cmake.in
+++ b/KF5AuthConfig.cmake.in
@@ -17,8 +17,8 @@ include("${CMAKE_CURRENT_LIST_DIR}/KF5AuthMacros.cmake")
 
 if(NOT "${KAUTH_BACKEND_NAME}" STREQUAL "FAKE")
     if(CMAKE_CROSSCOMPILING AND KF5_HOST_TOOLING)
-        find_file(TARGETSFILE KF5Auth/KF5AuthToolsTargets.cmake PATHS ${KF5_HOST_TOOLING} ${CMAKE_CURRENT_LIST_DIR} NO_DEFAULT_PATH)
-        include("${TARGETSFILE}")
+        find_file(KAUTH_TARGETSFILE KF5Auth/KF5AuthToolsTargets.cmake PATHS ${KF5_HOST_TOOLING} ${CMAKE_CURRENT_LIST_DIR} NO_DEFAULT_PATH)
+        include("${KAUTH_TARGETSFILE}")
     else()
         include("${CMAKE_CURRENT_LIST_DIR}/KF5AuthToolsTargets.cmake")
     endif()
